Src/BootCamp.Chapter/BmiCalculator.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using BootCamp.Chapter.Logger;
+
+namespace BootCamp.Chapter
+{
+    public static class BmiCalculator
+    {
+        private static readonly ILogger ConsoleLogger = new ConsoleLogger();
```

Comment:
With OOP, forget static. Static = single instance. You might want multiple instances of Bmi calculator.

Reply:
You say, call the function as an object?
Something like this?
```c#
var bmi = new BmiCalculator();
bmi.Calculate();
```